source/src/switch/cs_SwitchAggregator.cpp

```diff
@@ -361,20 +448,19 @@ void SwitchAggregator::executeStateIntentionUpdate(uint8_t value, cmd_source_wit
 			break;
 		}
 	}
+	printStates(__LINE__);
 
 	// Don't allow override reset in updateState, it has just been requested to be set to `value`
 	cs_ret_code_t retCode = updateState(false, source);
+	LOGSwitchAggregatorDebug("updateState retcode: %u", retCode);
+
+	printStates(__LINE__);
 
 	switch (retCode) {
-		case ERR_SUCCESS: {
-			[[fallthrough]];
-		}
-		case ERR_SUCCESS_NO_CHANGE: {
-			[[fallthrough]];
-		}
-		case ERR_NOT_POWERED: {
+		case ERR_SUCCESS:
+		case ERR_SUCCESS_NO_CHANGE:
+		case ERR_NOT_POWERED:
```

Comment:
Then I prefer to break them individually rather than falling through: the `{ [[fallthrough]] }` is more verbose, exotic, susceptible to failure than a normal `break` statement and doesn't even get syntax highlighted. :-)